plugins/modules/zos_blockinfile.py

```diff
@@ -382,6 +390,12 @@ def quotedString(string):
     return string.replace('"', "")
 
 
+def quotedString_outputJson(string):
```

Comment:
Also minor, try to conform to snake case, so `quotedString_outputJson` would be `quoted_string_output_json`

Reply:
Thanks for the snake case, looking back much of this code was written as camel case, oh well, we can change that some day, at least we started now. :)